crates/flox/src/commands/environment.rs

```diff
@@ -582,16 +594,32 @@ pub enum PushFloxmainOrEnv {
     },
 }
 
+#[derive(Debug, Clone, Bpaf)]
+enum PullSelect {
+    New {
+        /// Directory to create the environment in (default: current directory)
```

Comment:
Question: Is this saying that if you don't have a `.flox` and you do `flox pull some/env` it will create the `.flox` inside of this directory, or that it will use this directory _as_ `.flox` i.e. it will put `env.json` inside of this directory?

Reply:
it will create _a new_ `.flox` in that directory linked to `some/env`

currently we equal .flox with an environment (considering we might add flox config that might not always be true but for now is a good heuristic) so if there is a .flox present in `dir` we bail